bendigobank_spec.rb

```diff
@@ -1,19 +1,33 @@
 # frozen_string_literal: true
 
 require 'rspec'
-require 'Nokogiri'
+require 'json'
+require 'nokogiri'
 require_relative 'account'
 require_relative 'transaction'
 describe Account do
```

Comment:
It is only example. Write how you want.
Please read again guide for rspec https://rspec.info

You should run and check your spec `rspec bendigobank_spec.rb`
Here you describe class `Bendigobank` not `Account`.

Use data from html files, check parsing methods for parse accounts and transactions.
Show objects count and an object example in hash format.

Example:

```ruby
describe Bendigobank do
  html     = Nokogiri::HTML(File.read('your_accounts_file.html'))
  accounts = Bendigobank.parse_accounts(html)

  describe "#parse_accounts" do
    it 'show example for account object' do
      expect(accounts.count).to eq(5)
      expect(accounts.first.to_hash).to eq(
        {
          "name"         => "Demo Account",
          "currency"     => "USD",
          "balance"      => 100.50,
          "nature"       => "account",
          "transactions" => []
        }
      )
    end
  end
end
```

Do the same for transactions.